packages/playground/src/components/execution/key-value-table.tsx

```diff
@@ -25,13 +25,98 @@ interface KeyValueTableProps {
   tableSpec: KeyValuePair[]
 }
 
+//manage style for repeated use
+const accordionContentStyle = 'w-full pl-0 pr-0 border-0 pb-0'
+const specTitleStyle = 'text-ring flex-grow text-left -ml-1'
+
 export const KeyValueTable: React.FC<KeyValueTableProps> = ({
   className,
   tableTitleName,
   tableTitleVal,
   tableSpec
 }) => {
-  // ToDO:const getAccordian = () => <div></div> // do this function recursively
+  const renderListItems = (items: KeyValuePair[], level: number = 1) => {
+    //detect if the listItems is objects or array, tailwind css will not generate
+    const listItems = Array.isArray(items) ? items : [items]
+
+    return listItems.map((item, index: number) => {
+      if (typeof item.value === 'string') {
+        return (
+          <ul className="border-b" key={index}>
```

Comment:
ul and li usage here feels like an antipattern. why doesn't AccordionContent handle <Table>? Could the Accordion component be altered to allow for that?